### PR TITLE
[v1.x] ONNX 1.6 compatibility fix + fix for when multiple nodes have the same name

### PR DESCRIPTION
**Pull request by Zha0q1**
This pr fixes:
1. softmax when axis == -1
2. compatibility with onnx 1.6
3. adds a prefix (index) to the exported nodes so that we can handle when multiple nodes have the same name

### CODE REVIEW

**Review by josephevans — APPROVED** [review]
LGTM, thanks!

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 1]
```diff
@@ -2333,14 +2333,16 @@ def convert_layer_norm(node, **kwargs):
...
+    dtype = onnx.mapping.TENSOR_TYPE_TO_NP_TYPE[input_type]
 
     nodes = [
         create_tensor([axes], name+"_axes", kwargs["initializer"]),
         create_tensor([axes+1], name+"_axes+1", kwargs["initializer"]),
         create_tensor([], name+"_void", kwargs["initializer"]),
         create_const_scalar_node(name+'_0_s', np.int64(0), kwargs),
         create_const_scalar_node(name+'_1_s', np.int64(1), kwargs),
-        create_const_scalar_node(name+"_2_s", np.int64(2), kwargs),
+        create_const_scalar_node(name+"_2_s", np.array(2, dtype=dtype), kwargs),
```
Comment (waytrue17) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:2345]: Curious, why do we use array for this const node?

Comment (Zha0q1) [reply python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:2345]: This is a constant node. We just need to control the dtype so now we cannot use np.int64 or for that same reason np.float32 any more

Comment (waytrue17) [reply python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:2345]: Makes sense, thanks!

**Review by waytrue17 — APPROVED** [review]

**Pull request merged**